src/Vehicle/Vehicle.cc

```diff
@@ -1000,8 +1000,17 @@ void Vehicle::_chunkedStatusTextCompleted(uint8_t compId)
 
     // If the message is NOTIFY or higher severity, or starts with a '#',
     // then read it aloud.
-    if (messageText.startsWith("#") || severity <= MAV_SEVERITY_NOTICE) {
+    bool readAloud = false;
+
+    if (messageText.startsWith("#")) {
         messageText.remove("#");
```

Comment:
This isn't correct either. QString::remove removes all occurances of the specified character. This should only remove the first char from the string.

Reply:
True, I'll fix it.

Reply:
Done in 68b649e9a.